test/module/shared_model/builders/protobuf/transport_builder_test.cpp

```diff
@@ -203,10 +204,7 @@ class TransportBuilderTest : public ::testing::Test {
    * @param successCase function invoking if value exists
    * @param failCase function invoking when error returned
    */
-  template <typename SV,
-            typename T,
-            typename SuccessCase,
-            typename FailCase>
+  template <typename SV, typename T, typename SuccessCase, typename FailCase>
```

Comment:
Since you have touched that line, could you introduce meaningful names for template parameters SV and T?

Reply:
OK, I'll try)